python/dgl/distributed/__init__.py

```diff
@@ -17,6 +17,7 @@
 from .nn import *
 from .partition import (
     dgl_partition_to_graphbolt,
+    gb_convert_single_dgl_partition,
```

Comment:
Here's the arg list of original API, seems not exactly the same. we'd better keep it exactly same.
https://github.com/CfromBU/dgl/blob/be716ee8d766b0997a9dafca1d093e4c969b74fc/python/dgl/distributed/partition.py#L1823-L1830